build_scripts/maven_setup/setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Download and extract Maven
+wget http://mirror.jax.hugeserver.com/apache/maven/maven-3/3.5.0/binaries/apache-maven-3.5.0-bin.zip -P /tmp
```

Comment:
Can we add a flag to provide the tar file if you don't have an internet connection on the server?